README.md

```diff
@@ -2,14 +2,21 @@
 [![CircleCI Build Status](https://circleci.com/gh/martukas/nuclei.svg?style=shield)](https://circleci.com/gh/martukas/nuclei/tree/master)
 
 An Evaluated Nuclear Structure Data
-([ENSDF](https://www.nndc.bnl.gov/ensdf/))
+([ENSDF](https://www.nndc.bnl.gov/ensdf/)) 
 parser, viewer and editor.
 
 A future library for native ENSDF access and manipulation in C++.
 
+## Features
+
+Nuclei is a software tool for the displaying of nuclear decay schemes and estimated energy spectra, the calculation of angular γ emission anisotropies, and the automated search for appropriate decay cascade properties.
```

Comment:
I am uncertain if that's still true. In my modernization, I think I removed the sprectra generation feature.